server/util/live-loan/live-loan-fetch.js

```diff
@@ -60,8 +66,27 @@ async function fetchLoansFromGraphQL(request, resultPath) {
 }
 
 // Get per-user recommended loans from the ML service
-async function fetchRecommendationsByLoginId(id, flss = false) {
-	if (flss) {
+async function fetchRecommendationsByLoginId(id, queryType = QUERY_TYPE.DEFAULT) {
+	if (queryType === QUERY_TYPE.RECOMMENDATIONS) {
+		return fetchLoansFromGraphQL(
+			{
+				query: `query($userId: Int, $limit: Int) {
+					loanRecommendations(
+						userId: $userId,
+						limit: ${loanCount},
+						origin: "email:live-loans"
+					) {
+						${loanValues}
+					}
+				}`,
+				variables: {
+					userId: Number(id),
+					limit: loanCount
```

Comment:
@michelleinez @dyersituations There might be an issue here with `loanCount`...it's used below but not defined like this...

Reply:
Isn't it defined up above on line 11?  It also looks like it was originally used in this same exact way down below in the FLSS fetch query...

Reply:
I agree it appears to be! But you,ll want to confirm the actual query operation and combination of variables and that the failure isn't somewhere in the code setup...